src/pods/impl.c

```diff
@@ -0,0 +1,692 @@
+        }
+
+        if ((id_chk == 0) && (strlen(acc_id) > 0))
+        {
+                cpeabStrncpy(id_type,acc_id_ret,BFR_SIZE_64*sizeof(char));
+                ret = RETURN_OK;
+                return ret;
+        }
+        else
+        {
+                WebcfgError("Account ID couldnt be parsed\n");
+                return ret;
+        }
+
+}
+char * getPartnerID()
+{
+        int partner_id_chk = 1;
+        char *partner_id = NULL;
+        int ret_val;
+
+  	partner_id = malloc(BFR_SIZE_64*sizeof(char));
+        if (!partner_id)
+        {
+                WebcfgError("%s : partner_id couldnt be assigned with memory\n",__func__);
+                return NULL;
+        }
+        ret_val = get_id_pstore(partner_id_chk,partner_id);
+        if (ret_val != RETURN_ERR)
+        {
+                WebcfgDebug("PartnerID returned from lib is %s\n", partner_id);
+                return partner_id;
+        }
+        else
+        {
+                WebcfgError("Failed to GetValue for PartnerID\n");
+                return NULL;
+        }
+}
+
+char * getAccountID()
+{
+        int acc_id_chk = 0;
+        char *account_id = NULL;
+        int ret_val;
+
+        account_id = malloc(BFR_SIZE_64*sizeof(char));
+        if (!account_id)
+        {
+                WebcfgError("%s : account_id couldnt be assigned with memory\n",__func__);
+                return NULL;
+        }
+        ret_val = get_id_pstore(acc_id_chk,account_id);
+        if (ret_val != RETURN_ERR)
+        {
+                 WebcfgDebug("AccountID returned from lib is %s\n", account_id);
+                 return account_id;
+        }
+        else
+        {
+                WebcfgError("Failed to GetValue for AccountID\n");
+                return NULL;
+        }
+}
+
+char * getFirmwareUpgradeStartTime()
+{
+        char * start_time = NULL;
+
+	start_time = malloc(BFR_SIZE_32*sizeof(char));
+        if (!start_time)
+        {
+                WebcfgError("%s : start_time couldnt be assigned with memory\n",__func__);
+                return NULL;
+        }
+
+        cpeabStrncpy(start_time,"7200",strlen("7200")+1);
+        return start_time;
+}
+
+char * getFirmwareUpgradeEndTime()
+{
+        char * end_time = NULL;
+        
+	end_time = malloc(BFR_SIZE_32*sizeof(char));
+        if (!end_time)
+        {
+                WebcfgError("%s : end_time couldnt be assigned with memory\n",__func__);
+                return NULL;
+        }
+
+        cpeabStrncpy(end_time,"14400",strlen("14400")+1);
+        return end_time;
+}
+
+int Get_Webconfig_URL( char *pString)
+{
+        char url[128];
+        int ret = RETURN_ERR;
+
+        memset(url,0,sizeof(url));
+
+        if (json_string_value_get(WEBCFG_URL_PARAM,url, sizeof(url)))
+        {
+                 cpeabStrncpy(pString,url,sizeof(url));
+                 WebcfgDebug("Successfully fetched Webconfig URL : [%s]. \n", url);
+                 ret = RETURN_OK;
+        }
+        else{
```

Comment:
Please run the code formatting across the code so this is all normalized.  This will reduce maintenance efforts in the future.

Reply:
I realized I don't have the code format checking code in across the board.  I'm working on this & we can do it after this PR.